WebDriverAgentTests/IntegrationTests/FBAppiumTouchActionsIntegrationTests.m

```diff
@@ -211,7 +211,7 @@ - (void)testPress
   NSArray<NSDictionary<NSString *, id> *> *gesture =
   @[@{
       @"action": @"press",
-      @"options": @{
+      @"options": @{ // Here does not work in iOS 13
```

Comment:
I'll check this one later. Perhaps, something does not work as expected in iOS 13